test/test_integration_cluster.rb

```diff
@@ -42,7 +42,7 @@ def test_phased_restart_does_not_drop_connections
 
   def test_phased_restart_does_not_drop_connections_threads_fork_worker
     restart_does_not_drop_connections num_threads: 10, total_requests: 3_000,
-      signal: :USR1 #, config: 'fork_worker', log: true
+      signal: :USR1, config: 'fork_worker'
   end
```

Comment:
If this is the only test that needs `sleep 1` in `restart_does_not_drop_connections`, what about passing that in from the test? Making it more clear why we need to sleep 1 second

Or does it always make sense to sleep 1 second? I'm not sure what we're waiting for, that should probably be explained better in the comment above the sleep.

Reply:
> Or does it always make sense to sleep 1 second? I'm not sure what we're waiting for, that should probably be explained better in the comment above the sleep.

IMO it always makes sense to sleep 1 second. The sleep is just how long we wait before signalling another restart. All the tests that use `#restart_does_not_drop_connections` at most set a `total_requests` of 3000. With a sleep of 0.15 we restart ~10 times, so something like 300 requests before each restart. With a sleep of 1, ~3 times, so 1000 requests before each restart.

Seeing as the tests are to ensure connections aren't dropped:
1. I don't think the number of restarts really matters, as long as there's at least one
2. I don't think (hot/phased) restarting puma every 300 requests consecutively is a realistic scenario anyway

Essentially, this is just an arbitrary threshold that just so happens to time out one particular test, and I don't think adjusting it to the new value for all tests affects their accuracy.

Reply:
Agree, thanks for explaining!